### PR TITLE
update manifests for supporting accelerator UI

### PR DESCRIPTION
**Pull request by kvfasil**
## What
Modified manifest file with latest ripple contract
Fix for api's issue in Securestorage and metrics general processor.

## Why
Open source ripple fetching manifest file from Example folder.
Securestorage and metrics Tests failing.

## How
Updated extn-manifest with ripple contracts.
Added required changes in manifest files.
Added expected responses in Securestorage and metrics processor.

## Test
Manual test. 
Can be tested per usual manual testing procedure used by QA to test BI metrics.

## Checklist

- [x] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

File: examples/reference-manifest/IpStb/firebolt-extn-manifest.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        "browser",
+        "permissions",
+        "wifi",
+        "local.storage",
+        "remote_accessory",
+        "secure.storage",
+        "advertising",
+        "privacy_settings",
+        "metrics",
+        "root.session",
```
Comment (satlead) [inline examples/reference-manifest/IpStb/firebolt-extn-manifest.json:98]: Lot of cleanup needed in this section

Comment (satlead) [reply examples/reference-manifest/IpStb/firebolt-extn-manifest.json:98]:
```suggestion
        "required_contracts": [
        "rpc",
        "lifecycle_management",
        "device_info",
        "window_manager",
        "browser",
        "permissions",
        "wifi",
        "local.storage",
        "remote_accessory",
        "secure.storage",
        "advertising",
        "privacy_settings",
        "metrics",
        "account.session",
        "behavior_metrics",
}
```

File: examples/reference-manifest/IpStb/firebolt-device-manifest.json [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+      "xrn:firebolt:capability:protocol:dial",
+      "xrn:firebolt:capability:protocol:wifi",
+      "xrn:firebolt:capability:usergrant:pinchallenge",
+      "xrn:firebolt:capability:usergrant:acknowledgechallenge",
+      "xrn:firebolt:capability:input:keyboard",
+      "xrn:firebolt:capability:remote:ble",
+      "xrn:firebolt:capability:storage:secure"
+    ],
+    "dependencies": {
+      "xrn:firebolt:capability:privacy:advertising": [
```
Comment (satlead) [inline examples/reference-manifest/IpStb/firebolt-device-manifest.json:138]: Remove dependent capabilities

File: examples/reference-manifest/IpStb/firebolt-device-manifest.json [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+                }
+              ]
+            }
+          ],
+          "scope": "app",
+          "lifespan": "once",
+          "overridable": false
+        }
+      },
+      "xrn:firebolt:capability:account:id": {
```
Comment (satlead) [inline examples/reference-manifest/IpStb/firebolt-device-manifest.json:208]: Remove this

**Review by satlead — CHANGES_REQUESTED** [review]
Please fix the manifests

File: examples/reference-manifest/IpStb/firebolt-extn-manifest.json (hunk 1 above)
Comment (kvfasil) [reply examples/reference-manifest/IpStb/firebolt-extn-manifest.json]: done

File: examples/reference-manifest/IpStb/firebolt-extn-manifest.json [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        },
+        {
+            "path": "libdistributor_general",
+            "symbols": [
+                {
+                    "id": "ripple:channel:distributor:general",
+                    "uses": [
+                        "config"
+                    ],
+                    "fulfills": [
```
[the remark this answers is not in the document]
Comment (kvfasil) [reply examples/reference-manifest/IpStb/firebolt-extn-manifest.json:65]: done

File: examples/reference-manifest/IpStb/firebolt-device-manifest.json (hunk 3 above)
Comment (kvfasil) [reply examples/reference-manifest/IpStb/firebolt-device-manifest.json:136]: kept 3 grant policies.

File: examples/reference-manifest/IpStb/firebolt-app-library.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  "default_library": [
+    {
+      "app_id": "refui",
+      "manifest": {
+        "embedded": {
+          "app_key": "xrn:firebolt:application:refui",
+          "name": "refui",
+          "content_catalog": "refui_some_content_catalog",
+          "start_page": "http://127.0.0.1:50050/lxresui/index.html?__firebolt_endpoint=ws%3A%2F%2F127.0.0.1%3A3474%3FappId%3Drefui%26session%3Drefui#menu",
+          "runtime": "lightning",
```
[the remark this answers is not in the document]
Comment (kvfasil) [reply examples/reference-manifest/IpStb/firebolt-app-library.json:11]: yes

File: examples/reference-manifest/IpStb/firebolt-device-manifest.json [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+      "xrn:firebolt:capability:storage:secure"
+    ],
+    "grantPolicies": {
+      "xrn:firebolt:capability:account:id": {
+        "use": {
+          "options": [
+            {
+              "steps": [
+                {
+                  "capability": "xrn:firebolt:capability:usergrant:fingerprint"
```
Comment (satlead) [inline examples/reference-manifest/IpStb/firebolt-device-manifest.json:142]: dnt think we have support for finger print

Comment (kvfasil) [reply examples/reference-manifest/IpStb/firebolt-device-manifest.json]: removed mentioned feature.

**Review by satlead — APPROVED** [review]

**Review by Karthick-Somasundaresan — APPROVED** [review]
LGTM

**Review by joseinweb — APPROVED** [review]

**Review by Nv-Delle1416 — APPROVED** [review]

**Pull request merged**